tests/interfaces/test_batch_jax_qnode.py

```diff
@@ -1104,7 +1220,7 @@ def circuit(data, weights, coeffs):
     #     spy = mocker.spy(qml.transforms, "hamiltonian_expand")
     #     obs = [qml.PauliX(0), qml.PauliX(0) @ qml.PauliZ(1), qml.PauliZ(0) @ qml.PauliZ(1)]
 
-    #     @qnode(dev, interface="jax", diff_method=diff_method, mode=mode, max_diff=max_diff)
+    #     @qnode(dev, interface=interface, diff_method=diff_method, mode=mode, max_diff=max_diff)

```

Comment:
@antalszava it looks like this test can be uncommented now?

Reply:
Oh yes! It works for parameter-shift now, thanks :heart_eyes: